src/tests/super-admins.test.js

```diff
@@ -1 +1,162 @@
+      await superAdmins.collection.deleteMany({});
+      const response = await request(app).get('/api/super-admins').send();
+      expect(response.body.data).toBe(undefined);
+      expect(response.status).toBe(404);
+      expect(response.body.message).toMatch('Cannot find any Super Admin, please create one');
+    });
+  });
+});
+
+describe('GET BY ID /api/super-admins/:id', () => {
+  test('The database must return 200 if the id exist', async () => {
+    const response = await request(app).get(`/api/super-admins/${firstSuperAdminId}`).send();
+    expect(response.status).toBe(200);
+    expect(response.body).toBeDefined();
+    expect(response.body.data).toHaveProperty('_id', 'firstName', 'email', 'password');
+    expect(response.body.message).toMatch(`Super Admin Found! ${firstSuperAdminName}`);
+    expect(response.body.error).toBeFalsy();
+  });
+  test('The status must be 404, if the url is wrong', async () => {
+    const response = await request(app).get('/api/superAdmins').send();
+    expect(response.status).toBe(404);
+    expect(response.body.error).toBeUndefined();
+  });
+  test('The database must return 400 if the id is not a mongoose id, and said The ID is not valid', async () => {
+    const invalidId = '1';
+    const response = await request(app).get(`/api/super-admins/${invalidId}`).send();
+    expect(response.status).toBe(400);
+    expect(response.body.message).toMatch('The ID is not valid');
+    expect(response.body.error).toBeTruthy();
+  });
+  test('The database must return 404 if the id does not exist', async () => {
+    const nonExistentId = '678a4ffafc13ae184b753af0';
+    const response = await request(app).get(`/api/super-admins/${nonExistentId}`).send();
+    expect(response.status).toBe(404);
+    expect(response.body.message).toMatch('Super Admin not found with this id');
+    expect(response.body.error).toBeTruthy();
+  });
+});
+
+describe('POST /api/super-admins', () => {
+  test('The status must be 404, if the url is wrong', async () => {
+    const response = await request(app).post('/api/superAdmins').send();
+    expect(response.status).toBe(404);
+    expect(response.body.error).toBeUndefined();
+  });
+  test('When create an super admin status must be 201, and must have firstName, email and password defined', async () => {
+    const response = await request(app).post('/api/super-admins').send(mockSuperAdmin);
+    expect(response.status).toBe(201);
+    expect(response.body).toBeDefined();
+    expect(response.body.data).toHaveProperty('firstName', 'email', 'password');
+    expect(response.body.data.email).toMatch(/^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.+-]+\.com$/);
+    expect(response.body.message).toMatch('Super Admin Created!');
+    expect(response.body.error).toBeFalsy();
+  });
+  test('When create an super admin status must be 400 if name is not defined', async () => {
```

Comment:
There is a 'n' left over in "an".. the same happens in some following lines.. 
'a' instead of 'an'.